beetsplug/lyrics.py

```diff
@@ -659,26 +671,133 @@ def commands(self):
             help=u'print lyrics to console',
         )
         cmd.parser.add_option(
+            u'-r', u'--write-rst', dest='writerst',
+            action='store', default='.', metavar='directory',
+            help=u'write lyrics to given directory as RST files',
```

Comment:
changed. i changed *all* occurences of `rst` (except the `.rst` file extension), i hope this is what you meant. it has the nice side-effect of making the `self.rst` variable renamed to the appropriate `self.rest`.